client/my-sites/plans/current-plan/index.jsx

```diff
@@ -146,19 +143,22 @@ class CurrentPlan extends Component {
 
 export default connect(
 	( state, ownProps ) => {
-		const selectedSite = getSelectedSite( state ),
-			selectedSiteId = getSelectedSiteId( state );
+		const selectedSite = getSelectedSite( state );
+		const selectedSiteId = getSelectedSiteId( state );
+
+		const isJetpack = isJetpackSite( state, selectedSiteId );
+		const isAutomatedTransfer = isSiteAutomatedTransfer( state, selectedSiteId );
 
 		return {
 			selectedSite,
 			selectedSiteId,
 			context: ownProps.context,
 			currentPlan: getCurrentPlan( state, selectedSiteId ),
 			isExpiring: isCurrentPlanExpiring( state, selectedSiteId ),
-			isJetpack: isJetpackSite( state, selectedSiteId ),
+			shouldShowDomainWarnings: ! isJetpack || isAutomatedTransfer,
```

Comment:
That's what we want but maybe the naming is misleading. For any .com or auto transfer site we want to mount the `<DomainWarngins />` component.